styles/blocks/footer/footer.less

```
@@ -0,0 +1,106 @@
+.footer {
+    background-color: #212122;
+
+    &__social {
+        border: 1px solid transparent;
+
+        @media @screen-gt-s {
+            & {
+                float: left;
+                width: 50%;
+            }
+        }
+        @media @screen-gt-m {
+            & {
+                width: 33.3%;
+            }
+        }        
+    }
+
+    &__stores {
+        @media @screen-gt-s {
+            & {
+                margin-top: 30px;                
+                float: right;
+                width: 50%;
+                text-align:right;
+            }
+        }
+        @media @screen-gt-m {
+            & {
+                margin-top: 30px;                
+                float: left;
+                width: 33.3%;
+                text-align:right;
```

```suggestion
                text-align: right;
```

проверь по всему реквесту - формат такой должен быть ```свойство:``` ```пробел``` ```значение```